invenio_rdm_records/services/pids/providers/datacite.py

```diff
@@ -32,7 +52,7 @@ class DataCiteProvider(BaseProvider):
     only at PIDStore level.
     """
 
-    def __init__(self, name, client, pid_type="doi",
+    def __init__(self, client, name="datacite", pid_type="doi",
```

Comment:
food for thoughts: should name be a property instead that cannot be overridden via constructor?

```
class ..
    name = "datacite"

    def __init__ ....
```

Reply:
Agree on this one, I have changed the code to reflect it.